src/features_creator/features_creator.py

```diff
@@ -82,3 +82,40 @@ def calculate_standard_deviation(data, cols):
     ...
 
 
+def calculate_average(df, pattern):
+    """[summary]
```

Comment:
I don't think we need this `[summary]`

Reply:
yea, it meant to be function summary, I updated it